include/aie/Dialect/AIEX/IR/AIEX.td

```diff
@@ -34,8 +36,39 @@ def AIEX_Dialect : Dialect {
   let extraClassDeclaration = [{
   }];
 
+  let useDefaultTypePrinterParser = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// AIEX Types
+//===----------------------------------------------------------------------===//
+def AIEX_BlockFloatingPointType : TypeDef<AIEX_Dialect, "blockFloat", [
+    MemRefElementTypeInterface
+  ]> {
+  let summary = "AIEX type representing a block floating point type.";
+  let mnemonic = "bfp";
+  let description = [{
+    This is a type representing a block floating point.
+    It is meant to eventually be lowered into a standard type further down the pipeline.
+    It the meantime, it can be used for blocked fp related dataflow adaptations.
+    Available types are bfp16ebs8 and bfp16ebs16.
+  }];
+
+  let parameters = (
+    ins StringRefParameter<"">:$block_type
+  );
+
+  let assemblyFormat = "`<` $block_type `>`";
+
+  // Indicate that our type will not add additional verification to the parameters.
+  // Since this type needs information about the targetModel to be verified, 
+  // checks are done in the bfp lowering pass instead of the verifier.
+  let genVerifyDecl = 0;
```

Comment:
The type should be valid (or not) independent of TargetModel. It's use can be validated later on during some legalization or TargetModel based checks.